### ISSUE TITLE
Staging: VAOS Request Queue Endless Spinning Load when CC request has invalid NPI

### ISSUE THREAD

**Issue by JRRoof** [department-of-veterans-affairs/va.gov-team#50422]
## Description
When loading the VAOS Requests/Pending page an endless loading spinner is encountered when trying to fetch a Community Care request with and invalid NPI

When this happens, we want to recognize the condition and display an error message that pending requests were unable to be retrieved with directions on who to contact for assistance.  Design-UX team to provide exact language to use.  
 
12/6 Simi investigated after triage call and worked with Kay.  Determined this was a very edge case.  Can wait until after go-live.  All tickets will be fixed after we go live.  

Note there is another ticket related to this in the front end logged a while back regarding caching of NPI data.  This is why in Staging if you enter a new request. Then go to the request queue, view details of all the requests will display without the endless spinning load.   Provider data is being cached from the submitted request and so no endless spinning load. Refresh the page, select to view details and you will get endless spinning load.  

---

## Acceptance Criteria

Given a CC request with invalid NPI
When veteran tries to load details of the CC request with bad NPI
Then page loads an error message stating error and action to take instead of endless loading spinner
And page content matches the [copy doc](https://www.figma.com/file/7H71ZMIjN4aiO343Ihb87A/Alert-when-can't-display-CC-provider-Name?node-id=2%3A4&t=cW4N4lCofn6aHweI-0) 
And page design matches the design spec

Given 
When 
Then 

### Design Assets
All spec copy is FPO. Use copy in copy doc.

## Definition of Done
- [ ] All acceptance criteria are met
- [ ] Documentation is updated

---
## How to configure this issue
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by outerpress** [comment]
We can use this language to start, though we may need to modify once I look at it across all the possible states:

`We're sorry, we can't display your provider's information right now. Try again later, or contact your [VA facility](link to facility locator).`

@JRRoof @simiadebowale the ticket only mentions CC requests. Just to confirm, let me know which of the following could be affected by this error:

* Confirmed CC requests
* Canceled CC requests
* Confirmed CC appointments
* Past CC appointments
* Canceled CC appointments

**Comment by JRRoof** [comment]
The issue of not loading because of invalid NPI will be resolved when [#43195](https://github.com/department-of-veterans-affairs/va.gov-team/issues/43195) is complete.

**Comment by outerpress** [comment]
TLDR; [here's the design file with the updated CC details pages.](https://www.figma.com/file/7H71ZMIjN4aiO343Ihb87A/Alert-when-can't-display-CC-provider-Name?node-id=2%3A4)

@klawyer @JRRoof @ldelacosta feel free to add comments (either here or in the design file).

This issue affects CC details pages:
* CC Requests, including Confirmed, Pending, and Canceled states
* CC Appointments, including Upcoming, Past, and Canceled states
    * Also note the alert message placement when VAOS can't load the provider's address/contact information.

**Comment by JRRoof** [comment]
Two BE tickets related to this issue: [VAOSR-4877](https://issues.mobilehealth.va.gov/browse/VAOSR-4877) & [VAOSR-4963](https://issues.mobilehealth.va.gov/browse/VAOSR-4963)

**Comment by ldelacosta** [comment]
Since #43195 is completed, we can proceed with updating the alert message when the endpoint does not have any providers.

**Comment by simiadebowale** [comment]
@ldelacosta - This is ready for validation

**Comment by ajmagdub** [comment]
Verified that no calls are being made to the community care provider endpoint.

**Comment by ldelacosta** [comment]
Validated the ticket. No new issues reported.

**Comment by ldelacosta** [comment]
Added the spec in [VAOS GitHub](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/product/page-logic/community-care-flow-logic.md)
